.gitignore

```diff
@@ -2,5 +2,5 @@
 coverage
 jspm_packages
 node_modules
-public
-*.log
\ No newline at end of file
+dist
```

Comment:
lovely; for my info is more common `dist` or `lib`? I guess it depends on what you are building 😅